CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Release 2025-04-25
+
+### Amazon.Lambda.AspNetCoreServer.Hosting (1.7.5)
```

Comment:
I think PJ's change file was missing the entry for `Amazon.Lambda.AspNetCoreServer` which was also changed in his PR.

Reply:
Fixed